modules/configure-alb-cert/README.md

```diff
@@ -0,0 +1,50 @@
+# Module container_alb_cert
+
+This example is used to SSL certificate that you store in IBM Cloud Certificate Manager for an Ingress Application Load Balancer (ALB). For more information, about container ALB certificate, see [setting up Kubernetes Ingress](https://cloud.ibm.com/docs/containers?topic=containers-ingress-types).
+
+## Example Usage
+```
+provider "ibm" {
+}
+
+module "container_alb_cert" {
+  //Uncomment the following line to make the source point to registry level
+  //source = "terraform-ibm-modules/cluster/ibm//modules/configure-alb-cert"
+
+  source = "../../modules/configure-alb-cert"
+
+  cert_crn    = var.cert_crn
+  secret_name = var.secret_name
+  cluster_id  = var.cluster_id
```

Comment:
Follow the same naming of attributes if we already defined  in other sub modules...like cluster (not cluster_id)